frontend/src/component/feature/StrategyTypes/RolloutSlider/RolloutSlider.tsx

```diff
@@ -104,18 +157,18 @@ const RolloutSlider = ({
                                 Rollout percentage
                             </StyledHeader>
                             <Typography variant='body2'>
-                                The rollout percentage determines the proportion
-                                of users exposed to a feature. It's based on the
-                                MurmurHash of a user's unique identifier,
-                                normalized to a number between 1 and 100. If the
-                                normalized hash is less than or equal to the
-                                rollout percentage, the user sees the feature.
-                                This ensures a consistent, random distribution
-                                of the feature among users.
+                                T2he rollout percentage determines the
+                                proportion of users exposed to a feature. It's
+                                based on the MurmurHash of a user's unique
+                                identifier, normalized to a number between 1 and
+                                100. If the normalized hash is less than or
+                                equal to the rollout percentage, the user sees
+                                the feature. This ensures a consistent, random
+                                distribution of the feature among users.
                             </Typography>
 
                             <StyledSubheader variant='h3'>
-                                Stickiness
+                                Stickiness1
```

Comment:
Uh, what's with the numbers this time around? 😅 

```suggestion
                                Stickiness
```